.circleci/config.yml

```diff
@@ -107,21 +121,72 @@ jobs:
             AMO_BASE_URL=https://addons-dev.allizom.org               \
             ./bin/deploy.sh
 
+  deploy-stage:
+    <<: *deployment_container
+    steps:
+      - *attach_deploy_workspace
+      - *install_aws_cli
+      - run:
+          name: addons-blog deployment (stage)
+          command: |
+            pwd && ls -l
+            AWS_ACCESS_KEY_ID="${STAGE_AWS_ACCESS_KEY_ID}"            \
+            AWS_SECRET_ACCESS_KEY="${STAGE_AWS_SECRET_ACCESS_KEY}"    \
+            ADDONS_BLOG_BUCKET="${STAGE_ADDONS_BLOG_BUCKET}"          \
+            ADDONS_BLOG_BUCKET_PREFIX="${ADDONS_BLOG_BUCKET_PREFIX}"  \
+            AMO_BASE_URL=https://addons.allizom.org                   \
+            ./bin/deploy.sh
+
 workflows:
   version: 2
   default-workflow:
     jobs:
       - test-wptheme
       - test-blog
-      - build-blog:
+      # -dev
+      - build-blog-dev:
           requires:
             - test-blog
           filters:
             branches:
               only: main
       - deploy-dev:
           requires:
-            - build-blog
+            - build-blog-dev
+          filters:
+            branches:
+              only: main
+      # -stage
+      - build-blog-stage:
+          # Note that the tags filter is needed because "Circle CI does not
+          # run workflows for tags unless you explicitly specify tag filters.
+          # Additionally, if a job requires any other jobs (directly or
+          # indirectly), you must specify tag filters for those jobs.
+          filters:
+            tags:
+              only: /.+-stage/
+            branches:
+              ignore: /.*/
+      - deploy-stage:
+          requires:
+            - build-blog-stage
+          filters:
+            tags:
+              only: /.+-stage/
+            branches:
+              ignore: /.*/
+
+  autodeploy-stage:
+    triggers:
+      - schedule:
+          # Every 3 hours (but without `*/3` because it is NOT supported), see:
+          # https://circleci.com/docs/2.0/workflows/#scheduling-a-workflow
+          cron: "0 0,3,6,9,12,15,18,21 * * *"
```

Comment:
I think we should definitely consider what it would take to have a webhook at some point. If we run this for a long time this is going to use a lot of resources even if nothing has actually changed.